[2024-09-30T17:54:25.336+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-30T17:54:25.366+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_yellow_trip manual__2024-09-30T17:47:30.127854+00:00 [queued]>
[2024-09-30T17:54:25.380+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_yellow_trip manual__2024-09-30T17:47:30.127854+00:00 [queued]>
[2024-09-30T17:54:25.380+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2024-09-30T17:54:25.396+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_staging_layer.dbt_run_yellow_trip> on 2024-09-30 17:47:30.127854+00:00
[2024-09-30T17:54:25.404+0000] {standard_task_runner.py:63} INFO - Started process 420 to run task
[2024-09-30T17:54:25.407+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Taxi_trip_pipeline', 'dbt_staging_layer.dbt_run_yellow_trip', 'manual__2024-09-30T17:47:30.127854+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/taxi_trip.py', '--cfg-path', '/tmp/tmp5cn8wf9k']
[2024-09-30T17:54:25.409+0000] {standard_task_runner.py:91} INFO - Job 62: Subtask dbt_staging_layer.dbt_run_yellow_trip
[2024-09-30T17:54:25.462+0000] {task_command.py:426} INFO - Running <TaskInstance: Taxi_trip_pipeline.dbt_staging_layer.dbt_run_yellow_trip manual__2024-09-30T17:47:30.127854+00:00 [running]> on host 72cf0e8bafa8
[2024-09-30T17:54:25.549+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='Taxi_trip_pipeline' AIRFLOW_CTX_TASK_ID='dbt_staging_layer.dbt_run_yellow_trip' AIRFLOW_CTX_EXECUTION_DATE='2024-09-30T17:47:30.127854+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-30T17:47:30.127854+00:00'
[2024-09-30T17:54:25.550+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-30T17:54:25.566+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-30T17:54:25.567+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --profiles-dir /opt/***/DBT_Project  --project-dir /opt/***/dbt_trip --models yellow_trip']
[2024-09-30T17:54:25.583+0000] {subprocess.py:86} INFO - Output:
[2024-09-30T17:54:28.448+0000] {subprocess.py:93} INFO - [0m17:54:28  Running with dbt=1.8.0
[2024-09-30T17:54:28.804+0000] {subprocess.py:93} INFO - [0m17:54:28  Registered adapter: postgres=1.8.0
[2024-09-30T17:54:29.211+0000] {subprocess.py:93} INFO - [0m17:54:29  Unable to do partial parsing because of a version mismatch
[2024-09-30T17:54:31.469+0000] {subprocess.py:93} INFO - [0m17:54:31  [[33mWARNING[0m]: Deprecated functionality
[2024-09-30T17:54:31.469+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-09-30T17:54:31.469+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-09-30T17:54:31.470+0000] {subprocess.py:93} INFO - information.
[2024-09-30T17:54:31.782+0000] {subprocess.py:93} INFO - [0m17:54:31  Encountered an error:
[2024-09-30T17:54:31.782+0000] {subprocess.py:93} INFO - Compilation Error
[2024-09-30T17:54:31.783+0000] {subprocess.py:93} INFO -   Model 'model.fact_trip.zone_lookup' (models/Staging/zone_lookup.sql) depends on a source named 'staging.zone_lookup' which was not found
[2024-09-30T17:54:32.768+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-09-30T17:54:32.769+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-30T17:54:32.780+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-09-30T17:54:32.784+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Taxi_trip_pipeline, task_id=dbt_staging_layer.dbt_run_yellow_trip, run_id=manual__2024-09-30T17:47:30.127854+00:00, execution_date=20240930T174730, start_date=20240930T175425, end_date=20240930T175432
[2024-09-30T17:54:32.797+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 62 for task dbt_staging_layer.dbt_run_yellow_trip (Bash command failed. The command returned a non-zero exit code 2.; 420)
[2024-09-30T17:54:32.812+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-09-30T17:54:32.820+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
